feat: import Teams Javascript package

joshgummersall opened the pull request:
This PR re-homes the adaptive teams package (similar to the .NET change a week or two ago). I wasn't sure exactly how to attack the CI changes, hopefully @ryanlengel can assist with this.

#minor

peterinnesmsft commented on packages/Teams/js/src/actions/sendMESelectItemResponse.ts:
```diff
@@ -0,0 +1,105 @@
+/**
+ * @module botbuilder-dialogs-adaptive-teams
```
Do we need to update these module headers as well?

peterinnesmsft reviewed (changes requested):
I imagine we also need to update the Microsoft.Bot.Components.sln to update the directory location of the C# Teams component.

tracyboehrer commented:
@joshgummersall See #924 and #925.  If you don't have time to apply these changes, I'll get to it.

joshgummersall commented:
@tracyboehrer I need to verify the `commandId` changes work, otherwise, this should be ready to rock. I simplified a few things and added proper linting as well.

tracyboehrer commented:
@joshgummersall Eric submitted a PR that changed the menu's for the Teams items.  You should just be able to pick up the new schemas.

peterinnesmsft commented on package.json:
```diff
@@ -4,6 +4,7 @@
   "workspaces": [
     "generators/*",
     "packages/*",
+    "packages/*/js",
```
Do we need both `packages/*` and `packages/*/js`?

joshgummersall replied:
Yeah, it's possible we can revisit this when we restructure packages for proper multi-language development. It's possible, for example, that `packages/**` would work.

peterinnesmsft commented on packages/Teams/js/package.json:
```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@microsoft/bot-components-teams",
+  "author": "Microsoft Corp.",
```
In the other packages, we've used the author 'Bot Framework'. This feels like the more correct author we ought to be using. Do we need to get this aligned on the others? I don't know if there's any team-wide guidance on this.

joshgummersall replied:
I don't know how important this key is. In fact, it generally includes an email address. Perhaps something we can reconcile post GA.

peterinnesmsft commented on packages/Teams/js/package.json:
```diff
@@ -0,0 +1,69 @@
+  "version": "1.0.0",
+  "license": "MIT",
+  "keywords": [
+    "botbuilder",
```
We'll need to update the keywords so that it appears in Package Manager. At the very least, I think that entails having the msbot-component keyword. There are other ones that I believe Andrew can provide and give guidance on what to use.

joshgummersall replied:
@clearab can you verify what tags we need for components to show up in package manager?

peterinnesmsft commented on build/yaml/pipelines/teams-js.yml:
```diff
@@ -0,0 +1,20 @@
+name: $(Build.BuildId)
```
My initial thought with the addition of the Language variable and this file is that we then end up with a lot of largely duplicate files. I don't think I would change the approach we're taking here for right now, but I would probably add this to the list of things to review post-R13 and see how we feel about it.

joshgummersall replied:
Yeah, we should probably reconcile the structure of multi-language packages, then make the templates smart enough to look for `${LANGUAGE}/*` and run the appropriate commands.
